api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -73,9 +73,24 @@ type VirtualMachineImageStatus struct {
 	// Deprecated
 	PowerState    string `json:"powerState,omitempty"`
 
-	// SupportedGuestOS indicates whether the VirtualMachineImage's osType is supported
-	// by the hosts in the cluster and VM Service
-	SupportedGuestOS *bool  `json:"supportedGuestOS,omitempty"`
+	// ImageSupportedState indicates whether the VirtualMachineImage is supported by VM Service,
+	// A VirtualMachineImage is supported by VM Service if:
+	// - If OS Type is of Linux Family
```

Comment:
also indicates support on the hosts of the cluster.

Reply:
This is probably better expressed in terms of conditions are true or not

Reply:
Yes agreed, the conditions have a detailed description of each of these cases, This is the summary field that we discussed in the call the other day, and this documentation useful to understand when this boolean would be set

Reply:
We are restating the conditions that need to be true. These descriptions are going to drift, and it already is because the `OsTypeSupportedCondition` condition doesn't say anything about "If OS Type is supported by hosts in the cluster". Express this in the conditions that need to be satisfied because this field is a summary of those.

Reply:
Fixed